build.gradle

```diff
@@ -3,9 +3,11 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.jakewharton:butterknife-gradle-plugin:8.6.0'
```

Comment:
In the documentation it was included. Is it running without this file? Is the Travis check passing? If yes, then not necessary.